src/test/java/com/example/springboot/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,58 @@
+package com.example.springboot.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import com.example.springboot.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.Assert;
+import org.junit.rules.ExpectedException;
+
+class FileReaderServiceImplTest {
+    public final ExpectedException exception = ExpectedException.none();
+    private static final String EMPTY_FILE_NAME = "test1";
+    private static final String CSV_FILE_NAME = "test2.csv";
+    private static final String CSV_FILE_NAME2 = "test3.csv";
```

Comment:
this constant variable name is not informative

Reply:
Done